metals/src/main/scala/scala/meta/internal/metals/Messages.scala

```diff
@@ -18,6 +18,12 @@ import org.eclipse.lsp4j.ShowMessageRequestParams
  */
 object Messages {
 
+  val noRoot = new MessageParams(
+    MessageType.Error,
+    """|No rootUri or rootPath detected.
```

Comment:
`No workspace folder detected` ?

`rootUri` and `rootPath` seem internal to LSP, so might be confusing to users

Reply:
So I guess I was purposefully avoiding that term since `workspaceFolder` is part of the LSP spec that we don't support. So I was trying to avoid using the term. Do you still think it may be clearer to use it?

Reply:
Right, it might be fine then with the current message. I was mostly thinking aloud here.